### DISCUSSION TITLE
Is it possible to run MediaCMS from behind some other Reverse Proxy

### DISCUSSION THREAD

**Discussion by Ascar699** [mediacms-io/mediacms#687] [Q&A]
Hello everyone, hello @mgogoulos 

I spent last few days trying to deploy on Docker with HAProxy on pFsense firewall acting as a reverse proxy.  I tried an uncountable number of configuration modifications but was not able to make it past accessing MediaCMS on local IP address of the webservice container.  Every re-configuration attempt ends with 503 No Server Available error.

Can you share with me what modifications to the compose file I should make in order to get MediaCMS working?

1. I made changes to the local_settings.py file and tried both http and https-prepended FQDNs for my subdomain
2. I tried modifying the docker-compose.xxx.yaml files with commenting and uncommenting the nginx-proxy service
3. Tried changes to the web service ports

I apparently get something done wrong.  I searched discussions for any mentions of HAProxy but it seems this is not widely used with MediaCMS?
Thanks for your help in advance

**Comment by Mozitux** [comment]
Yes, it's possible.
See: https://github.com/mediacms-io/mediacms/issues/500#event-7909730113
